receiver/kafkareceiver/unmarshaler.go

```diff
@@ -67,19 +69,23 @@ func defaultTracesUnmarshalers() map[string]TracesUnmarshaler {
 
 func defaultMetricsUnmarshalers() map[string]MetricsUnmarshaler {
 	otlpPb := newPdataMetricsUnmarshaler(&pmetric.ProtoUnmarshaler{}, defaultEncoding)
+	otlpJSON := newOTLPJSONMetricsUnmarshaler()
 	return map[string]MetricsUnmarshaler{
 		otlpPb.Encoding(): otlpPb,
+		otlpJSON.Encoding(): otlpJSON,
 	}
 }
 
 func defaultLogsUnmarshalers(version string, logger *zap.Logger) map[string]LogsUnmarshaler {
 	azureResourceLogs := newAzureResourceLogsUnmarshaler(version, logger)
+	otlpJSON := newOTLPJSONLogsUnmarshaler()
```

Comment:
move to line after otlpPb, for consistency

Reply:
understood